Lib/test/test_subprocess.py

```diff
@@ -2048,6 +2048,8 @@ def test_group_error(self):
         with self.assertRaises(ValueError):
             subprocess.check_call(ZERO_RETURN_CMD, group=65535)
 
+    # TODO: RUSTPYTHON, SpamError: not enough spam
+    @unittest.expectedFailure
```

Comment:
I put `SpamError: not enough spam` as a placeholder for you to provide the actual failure reason.

Reply:
Is this an accurate summary? Feel free to amend it.

```suggestion
    # TODO: RUSTPYTHON, observed gids do not match expected gids
    @unittest.expectedFailure
```

Reply:
As far as I currently understand, yes
Thanks for clearing it up!